packages/host/app/components/operator-mode/container.gts

```diff
@@ -532,7 +536,7 @@ export default class OperatorModeContainer extends Component<Signature> {
     ) {
       return htmlSafe(`background-image: url(${this.backgroundImageURLs[0]});`);
     }
-    return '';
```

Comment:
I can’t explain it but removing this brings back the backgrounds:

<img width="1550" alt="Boxel 2023-08-17 19-16-39" src="https://github.com/cardstack/boxel/assets/43280/74ab68d0-bf4f-428f-bd27-c9f924f856e6">